src/System.Net.Security/src/System/Net/NegotiateStreamPal.Unix.cs

```diff
@@ -134,7 +213,32 @@ internal static Exception CreateExceptionFromError(SecurityStatusPal statusCode)
             out int newOffset,
             uint sequenceNumber)
         {
-            throw new PlatformNotSupportedException();
+            if (offset < 0 || offset > (buffer == null ? 0 : buffer.Length))
```

Comment:
Right.. I meant that same fix will be needed in https://github.com/dotnet/corefx/blob/dev/negotiatestream/src/System.Net.Security/src/System/Net/NegotiateStreamPal.Windows.cs#L368